lib/src/main/java/org/team100/lib/motor/drive/Falcon6DriveMotor.java

```diff
@@ -31,39 +31,37 @@ public class Falcon6DriveMotor implements MotorWithEncoder100<Distance100> {
      * The speed, below which, static friction applies, in motor revolutions per
      * second.
      */
-    private static final double staticFrictionSpeedLimitRev_S = 3.5;
+    private static final double staticFrictionSpeedLimitRev_S = 0.1;
```

Comment:
i'm pretty sure the large value that used to be here was tuned on the ground, and i think you tuned this on blocks.  also this is affected by the other bug about gear ratio, see below.